crawler/plugins/disk_vm_crawler.py

```diff
@@ -0,0 +1,39 @@
+try:
+    from crawler.icrawl_plugin import IVMCrawler
+except ImportError:
+    from icrawl_plugin import IVMCrawler
+
+import logging
+
+try:
+    import psvmi
+except ImportError:
+    psvmi = None
+
+logger = logging.getLogger('crawlutils')
+
+
+class disk_vm_crawler(IVMCrawler):
+
+    def get_feature(self):
+        return 'disk'
+
+    def crawl(self, vm_desc, **kwargs):
```

Comment:
psvmi does not extract disk partition info. This should raise NotImplementedError()